python/lsst/ts/observatory/control/script_queue.py

```diff
@@ -361,7 +367,7 @@ async def list_standard_scripts(self) -> typing.List[str]:
 
         available_scripts = await self._get_available_scripts()
 
-        return available_scripts.standard.split(",")
+        return available_scripts.standard.split(":")
```

Comment:
Maybe extract this to a constant and use it wherever it needs to be used (for instance here and in line 383).